README.md

```diff
@@ -7,39 +7,50 @@ var TalonOne = require('talon-one')
 
 var applicationId = 15025
 var applicationKey = '41d3f05e76fd667b'
-
 var client = new TalonOne.IntegrationClient('https://mycompany.talon.one', applicationId, applicationKey)
 
+var sessionId = 'some-identifier-for-this-session'
 var customerId = 'id-used-by-my-company'
 
+
 client.updateCustomerProfile(customerId, {
-  // only include properties you want to update, null values are ignored
-  name: 'Val Kust',
+  attributes: {
+    // only include properties you want to update, null values are ignored
+    Name: 'Val Kust',
+  }
 }, function (err, integrationState) {
-  console.log(integrationState.profile)
-  console.log(integrationState.event)
+  console.log(integrationState) // session, profile, event information
 })
 
-var sessionId = 'some-identifier-for-this-session'
 
-client.updateCustomerSession(customerId, {
+
+client.updateCustomerSession(sessionId, {
   // associate this session with the profile we created above
-  profileId: profileId,
-  // set the currency being used in this session
-  currency: 'USD',
+  profileId: customerId,
+  // set referral ID for this session
+  referral: 'somereferral-identifier',
```

Comment:
Would this be something internal to the shopsystem or a referral code from our database?

Reply:
this value is stored on the `referral` field in the customer_sessions table. I am not sure, if that is a referral code from our DB or something else...